### ISSUE TITLE
TypeError: torch._C._LegacyVariableBase.__new__(Parameter) is not safe

### ISSUE THREAD

**Issue by jahuth** [jahuth/convis#3]
On PyTorch > 0.3, the creation of Variables throws a `TypeError` (confirmed by two users so far).

```python
>>> retina = convis.retina.Retina()
Traceback (most recent call last):

  File "<ipython-input-6-2f57ffb3d514>", line 1, in <module>
    retina = convis.retina.Retina()

  File "/usr/local/lib/python2.7/dist-packages/convis/retina.py", line 82, in __init__
    self.opl = rf.OPL()

  File "/usr/local/lib/python2.7/dist-packages/convis/filters/retina.py", line 482, in __init__
    self.opl_filter = HalfRecursiveOPLFilter()

  File "/usr/local/lib/python2.7/dist-packages/convis/filters/retina.py", line 243, in __init__
    self.relative_weight = Parameter(0.5,retina_config_key='opl-relative-weight')

  File "/usr/local/lib/python2.7/dist-packages/convis/variables.py", line 134, in __new__
    return super(Parameter, self).__new__(self,torch.Tensor(x))

  File "/usr/local/lib/python2.7/dist-packages/convis/variables.py", line 119, in __new__
    return super(Variable, self).__new__(self,torch.Tensor(x))

TypeError: torch._C._LegacyVariableBase.__new__(Parameter) is not safe, use torch._C._TensorBase.__new__()
```

**Comment by jahuth** [comment]
This is because "Tensors and Variables have merged", see [0.4.0 Migration guide](https://pytorch.org/2018/04/22/0_4_0-migration-guide.html).

Convis will support the current version of PyTorch after the next minor release `0.6` (coming very soon), but will have to break the functionality with the older versions.

**Comment by jahuth** [comment]
The current release `convis 0.6` should fix this issue.
